src/dom/createelement.js

```diff
@@ -26,7 +26,8 @@ import { isString } from 'lodash-es';
  * @returns {Element} Created element.
  */
 export default function createElement( doc, name, attributes = {}, children = [] ) {
-	const element = doc.createElement( name );
+	const namespace = attributes instanceof Object ? attributes.xmlns : null;
+	const element = namespace ? doc.createElementNS( namespace, name ) : doc.createElement( name );
```

Comment:
I think that you don't need to check whether `attributes` is an object. You can simply assume it is (which happens later in this function anyway).

Reply:
> You can simply assume it is (which happens later in this function anyway).

Actually, I thought there's `for-of` loop next, which would verify that `attributes` is an object. But it's `for-in` so it'll work fine with `undefined` and `null`. So, if other tests or the reality requires that you handle `attributes` being `null` or `undefined, then this should be fine:

```js
const namespace = attributes && attributes.xmlns;
```

Reply:
I actually had the problem that in some tests attributes was null and my code failed.

Reply:
So based on that, should I still change something or leave it like that?

Reply:
I think you could simplify this to:

```js
const namespace = attributes && attributes.xmlns;
```

I'll try this on `master` after merging your PR.